scapy/layers/ipsec.py

```diff
@@ -47,8 +47,17 @@
 from scapy.config import conf, crypto_validator
 from scapy.data import IP_PROTOS
 from scapy.error import log_loading
-from scapy.fields import (ByteEnumField, ByteField, StrField, StrLenField,
-                          XIntField, IntField, ShortField, PacketField)
+from scapy.fields import (
+    ByteEnumField,
+    ByteField,
+    IntField,
+    PacketField,
+    ShortField,
+    StrField,
+    XIntField,
+    XStrField,
+    XStrLenField,
+)
```

Comment:
This is quite ugly isn't it ? I guess removing the parenthesis + using only 2 lines would be easier to read